src/ResourceManager/Compute/Commands.Compute/Models/PSVirtualMachine.cs

```diff
@@ -88,34 +91,7 @@ public string ResourceGroupName
         // Gets or sets the storage profile.
         public StorageProfile StorageProfile { get; set; }
 
-        [JsonIgnore]
```

Comment:
This is unnecessary.  This property is just a copy of a nested property, which was used for piping scenario, but now the piping scenario works without these properties.  There was a complain about these redundant property.